setup.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+import codecs
+import os
+from typing import List
+
+from setuptools import setup
+
+
+def read(fname):
+    file_path = os.path.join(os.path.dirname(__file__), fname)
+    return codecs.open(file_path, encoding="utf-8").read()
+
+
+install_requires: List[List[str]] = []
+install_requires += (["jsonschema"],)
+
+
+setup(
+    name="ngff",
+    version="0.0.4dev",
```

Comment:
This `version` probably expresses best my concerns about the multi-type repository. Turning it into a question, what is our positions on the different versions in this repository? I see at least 3 sources of truth:"

- the version of the Python library shipping the schemas (0.0.4.dev)
- the version of the latest schema (currently 0.3.0)
- the latest repository tag (currently 0.1.3)
Are these versions expected to be in sync or evolve independently. For all the versions that are in sync, can we use a single-source of truth.

Reply:
The "version of the latest schema" is defined by the most recent top-level directory, currently `/0.3/` but I think that doesn't have to match the current version in setup.py, since the directory will never be named `*dev`. But at each release they should match and correspond to the repo tag (with bumpversion being the single source of truth).